js/watch/README.md

```
@@ -70,7 +70,7 @@ The simplest way to watch a stream:
 | `name`    | string                              | required | Broadcast name/path                      |
 | `paused`  | boolean                             | false    | Pause playback                           |
 | `muted`   | boolean                             | false    | Mute audio                               |
-| `visible` | `never` \| distance \| `always`     | `0px`    | When to download video (see below)       |
+| `visible` | `never` | distance | `always`     | `0px`    | When to download video (see below)       |
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 4; Actual: 6; Too many cells, extra data will be missing

(MD056, table-column-count)